settings-system.lp

```diff
@@ -70,67 +70,67 @@ mg.include('scripts/pi-hole/lua/settings_header.lp','r')
                     <div class="col-lg-12">
                         <table class="table table-striped table-bordered nowrap">
                             <tbody>
-                                <tr>
+                                <tr title="Client broadcast to locate available servers">
                                     <th scope="row">
                                         <span>DHCPDISCOVER:</span>
                                     </th>
                                     <td id="sysinfo-dhcp-discover">&nbsp;</td>
                                 </tr>
-                                <tr>
+                                <tr title="Server to client in response to DHCPDISCOVER with offer of configuration parameters">
                                     <th scope="row">
                                         <span>DHCPOFFER:</span>
                                     </th>
                                     <td id="sysinfo-dhcp-offer">&nbsp;</td>
                                 </tr>
-                                <tr>
+                                <tr title="Client message to servers either (a) requesting offered parameters from one server and implicitly declining offers from all others, (b) confirming correctness of previously allocated address after, e.g., system reboot, or (c) extending the lease on a particular network address">
```

Comment:
It's approved, but I have another small suggestion:

```suggestion
                                <tr title="Client message to servers either &#010;  (a) requesting offered parameters from one server and implicitly declining offers from all others, &#010;  (b) confirming correctness of previously allocated address after, e.g., system reboot, or &#010;  (c) extending the lease on a particular network address">
```

![image](https://github.com/pi-hole/web/assets/1385443/78057fae-8fae-48fd-8f20-aad20521a9a6)

Reply:
It looks less nice on narrow-screens with line-breaks in the individual items themselves but it is still better readable than one mega-block of text